model/metrics/process-metrics.yaml

```diff
@@ -1,21 +1,4 @@
 groups:
-  - id: attributes.process.cpu
```

Comment:
I think this part should stay here and we are moving into registry only attributes i.e. `state`